### DISCUSSION TITLE
Batterie im Winter über Tibber mit Huawei laden?

### DISCUSSION THREAD

**Discussion by Hutzelbrot** [hoylabs/OpenDTU-OnBattery#1097] [Ideas]
Vielleicht stelle ich mir das zu einfach vor: Man könnte doch bei fehlender Sonne die Batterie während einer günstigen Stunde mit Tibber Strom laden. Voraussetzung wäre, dass während des Ladens der Hoymiles ausgeschaltet ist und die dann arbeitslosen Victons die Spannung vom Netzteil abkönnen. Steuern könnte man das über Home-Assistant, da gibt es eine Integration für den Strompreis https://github.com/custom-components/nordpool und eine Automation für die günstigste(n) Stunde(n) https://www.creatingsmarthome.com/index.php/2022/09/17/home-assistant-nord-pool-spot-prices-and-how-to-automate-devices-for-cheapest-hours/

**Comment by spcqike** [comment]
Steuern kannst du das auch mit EVCC. 

Was genau ist die Frage? Ob es möglich ist? Ja. Aber nicht mit openDTU allein. Ob es sinnvoll ist? Ein klares „kommt darauf an“

**Reply by Hutzelbrot** [reply]
Die Fragen: vertragen Victrons Fremdspannung am Ausgang? ist innerhalb openDTU-OnBattery eine Einstellung Huawei On Hoymiles Off möglich?

**Reply by spcqike** [reply]
> vertragen Victrons Fremdspannung am Ausgang?

Am Batterie-Ladeausgang? Würde ich von ausgehen. Immerhin kann man auch mehrere Victron parallel schalten, auch unterschiedliche. AC und MPPT. Solange die Spannung passt. Aber das sollte sie so oder so, sonst leidet die Batterie. 

> ist innerhalb openDTU-OnBattery eine Einstellung Huawei On Hoymiles Off möglich?

Ob es dafür eine Art Pre-Set gibt, weiß ich nicht. Allerdings meine ich hier in diversen issues gelesen zu haben, dass der AC charger auf eine gewisse Spannung / SoC laden kann, bevor der hoymiles durch den DPL gestartet wird. Also ja: irgendwie wird es gehen. Notfalls manuell. Den DPL kann man notfalls per MQTT steuern. Eine externe, übergeordnete Steuerung brauchst du so oder so (bisher)
